frontend-react/e2e/helpers/external-links.ts

```diff
@@ -15,8 +15,6 @@ export class ExternalLinks {
         await expect(newTab).toHaveURL(
             "https://app.smartsheetgov.com/b/form/48f580abb9b440549b1a9cf996ba6957",
         );
-        await expect(
-            newTab.getByText("Connect with ReportStream"),
-        ).toBeTruthy();
+        expect(newTab.getByText("Connect with ReportStream")).toBeTruthy();
```

Comment:
above `expect` needs to have `await` and here it doesn't?

Reply:
It varies per-method. The getByText method is synchronous and with the new activated lint rules gets properly called out. It would need to be refactored to use an async method if that becomes an issue.